WordPress/src/main/java/org/wordpress/android/widgets/WPNetworkImageView.java

```diff
@@ -159,6 +160,23 @@ private void loadImageIfNecessary(final boolean isInLayoutPass,
             return;
         }
 
+        int width = getWidth();
```

Comment:
> wrong image from appearing when used in lists

Are we still having this problem? I think we've fixed it by adding a reference to the requested URL, and match the URL `onResponse`.